go/felix/iptables/table.go

```diff
@@ -531,6 +533,14 @@ func (t *Table) Apply() {
 				failedAtLeastOnce = true
 				continue
 			} else {
+				t.logCxt.WithError(err).Error("Failed to program iptables, loading diags before panic.")
+				cmd := t.newCmd(t.iptablesSaveCmd, "-t", t.Name)
+				output, err2 := cmd.Output()
+				if err2 != nil {
+					t.logCxt.WithError(err2).Error("Failed to load iptables state")
+				} else {
+					t.logCxt.WithField("iptablesState", string(output)).Error("Current state of iptables")
+				}
```

Comment:
Definitely worse if I'm reading the logging and trying to understand why the programming failed.

But anyway, it's an orthogonal detail that we can come back to (if/when we care enough).